config/build.ts

```diff
@@ -40,31 +39,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
```

Comment:
Just be sure to check in the "prod" script lab that your new samples are showing up, and at the same time that everything else looks ok (i.e., that the groups are in the right order, and that the snippets within are in the right order as well.  A 20-second smoke-test).